Parsers/Creator Toolbox.js

```diff
@@ -0,0 +1,10 @@
+/*
+activation_example:!ctb
+regex:(!ctb)||(!creatortoolbox)||(!toolbox) 
+flags:gmi
+*/
+var sentence = current.text;
+
+
+var message = '<https://www.youtube.com/watch?v=fDvEqAQIHzQ&list=PL3rNcyAiDYK08_Xjuvu6IFEK8pqFT99WQ>';
```

Comment:
Please add text to the link as per the issue description:
The latest episode of Creator Toolbox

Reply:
@earlduque I've added the description